admin-dev/themes/new-theme/scss/pages/_modules.scss

```diff
@@ -611,30 +683,51 @@ form#importDropzone {
     text-align: center;
     vertical-align: middle;
     img {
-      max-width: 57px;
+      max-width: 40px;
```

Comment:
Please use rems instead of pixels if you can.

Reply:
:ok_hand: